lib/routes/account.js

```diff
@@ -1062,14 +1149,44 @@ module.exports = function (
                 resume: request.payload.resume,
                 acceptLanguage: request.app.acceptLanguage
               }
+            ))
+            .done(
+              function () {
+                reply({})
+              },
+              reply
             )
-          )
-          .done(
-            function () {
-              reply({})
-            },
-            reply
-          )
+
+        } else {
+          return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
Likewise here, can we not use the auth-strategy that handles this lookup for free?